packages/expect/src/matchers.ts

```diff
@@ -120,9 +120,9 @@ const matchers: MatchersObject = {
 
   toBeCloseTo(
     this: MatcherState,
-    received: number,
-    expected: number,
-    precision: number = 2,
+    received: number | bigint,
+    expected: number | bigint,
+    precision: number = NaN,
```

Comment:
Instead of default value, please assign `arguments.length === 3` to a variable like `hasPrecision`

Reply:
Would you want me to create an overload to keep params in intellisense ?

Reply:
nvm, misinterpreted at first, will do.

Reply:
Btw, this will make us have to recast the type (`precision as number`) for ts.

Reply:
pushed